llvm/docs/FatLTO.rst

```diff
@@ -76,3 +76,40 @@ Supported File Formats
 The current implementation only supports ELF files. At time of writing, it is
 unclear if it will be useful to support other object file formats like ``COFF``
 or ``Mach-O``.
+
+Usage
+===========
+
+Users are expected to pass ``-ffat-lto-objects`` to clang in addition to one of
+the ``-flto`` variants. Without the ``-flto`` flag, ``-ffat-lto-objects`` has
+no effect.
+
+Compile an object file using FatLTO:
+
+.. code-block:: console
+
+   $ clang -flto -ffat-lto-objects example.c -o example.o
+
+Link using the object code from the fat object without LTO:
+
+.. code-block:: console
+
+   $ clang -fno-lto -ffat-lto-objects -fuse-ld=lld example.o
```

Comment:
The description can state that `-ffat-lto-objects` is a no-op for linking when `-fno-lto` is specified.

Reply:
done.